policycoreutils: semanage

When modifying selinux login records, self.sename might not be
needed, but it IS passed to a log function, so it must be set
to something or the log attempt will fail.

Upstream-Status: Pending

Signed-off-by: Joe Slater <jslater@windriver.com>


--- a/semanage/seobject.py
+++ b/semanage/seobject.py
@@ -576,6 +576,7 @@ class loginRecords(semanageRecords):
 
 		if sename != "":
 			semanage_seuser_set_sename(self.sh, u, sename)
+			self.sename = sename
 		else:
 			self.sename = self.oldsename
 			
